src/applications/static-pages/covid-vaccine-updates-cta/widget.jsx

```diff
@@ -12,12 +12,16 @@ const copy = {
     appTitle: 'Covid 19 Vaccination Information',
     cta: `Sign up for an easy way to stay informed about getting a COVID-19
       vaccine at VA.`,
-    body: `   When you sign up, we’ll also ask about your vaccine plans. Your
-      local VA health facility may use this information to determine
-      when to contact you once your risk group becomes eligible.`,
-    boldedNote: `Note:`,
-    note: `We’ll contact every eligible Veteran in
-      each risk group. You don't need to sign up to get a vaccine`,
+    expandedEligibilityContent: {
+      veteran: {
+        boldedNote: `If you're a Veteran currently receiving care through VA,`,
+        body: ` we'll ask about your vaccine plans. Your local VA health facility may use this information to determine when to contact you once your risk group becomes eligible.`,
+      },
+      nonVeteran: {
+        boldedNote: `If you're a Veteran, spouse, or caregiver not receiving care through VA,`,
+        body: ` sign up to tell us if you want to get a vaccine through VA. If you're eligible, we'll contact you when we have a vaccine available for you. At this time, we don't know when that will be.`,
+      },
+    },
```

Comment:
Not necessary but I mentioned in an old code review for the Spanish content that instead of storing the language values in an object with named properties, it might be easier to just store the translated blocks as their own JSX components, in case we start seeing the block structures diverge between languages. That way the component is a little easier to read, versus using property names.

Reply:
agree. it took all of my 11pm friday evening brainpower to come up with a structure that formatted the new english content correctly but still rendered the Spanish and Tagalog content as it was. would have been much easier if each language was isolated and not dependent on a common component to render them. i suspect we will continue to be in a place (unfortunately) for a while where the English version of things goes out first and is (hopefully quickly) followed by updated Spanish and Tagalog versions. isolation would make this much easier.

Reply:
Haha I hadn’t even thought about the difficulty around naming each property on top of all that. Well said!!